client/my-sites/themes/theme-showcase.jsx

```diff
@@ -663,9 +659,6 @@ class ThemeShowcase extends Component {
 					isSiteECommerceFreeTrial={ isSiteECommerceFreeTrial }
 				/>
 				{ this.renderSiteAssemblerSelectorModal() }
-				{ isLoggedIn && (
-					<ThemeShowcaseSurvey condition={ () => previousPath.includes( 'theme/' ) } />
```

Comment:
This never renders anything because `<ThemeShowcaseSurvey />` expects a `survey` prop and renders nothing without it:

https://github.com/Automattic/wp-calypso/blob/7c12dde00bc229f739fe071aff49b0a33e28319d/client/my-sites/themes/survey/index.tsx#L58-L60

Reply:
Yeah we were directed to stop doing the surveys for a while and so the survey was purposefully disabled this way.

I agree, it's past time though, nice one for finding it.